test/viewmodels/GuaranteeAccountViewModelSpec.scala

```diff
@@ -18,21 +18,44 @@ package viewmodels
 
 import config.AppConfig
 import connectors.AccountStatusOpen
-import models.{GeneralGuaranteeBalance, GuaranteeAccount}
+import models.{Amounts, DueDate, GeneralGuaranteeBalance, GuaranteeAccount, GuaranteeTransaction, TaxType, TaxTypeGroup}
 import play.api.test.Helpers
 import utils.SpecBase
 
-import java.time.LocalDateTime
+import java.time.{LocalDate, LocalDateTime, Month}
 
-class GuaranteeAccountViewModelSpec extends SpecBase  {
+class GuaranteeAccountViewModelSpec extends SpecBase {
   val guaranteeAccount = GuaranteeAccount("gan", "eori", AccountStatusOpen,
-                                                 Some(GeneralGuaranteeBalance(BigDecimal(10000.50), BigDecimal(5000.10))))
+    Some(GeneralGuaranteeBalance(BigDecimal(10000.50), BigDecimal(5000.10))))
+
+  val guaranteeAccount02 = GuaranteeAccount("gan", "eori", AccountStatusOpen,
+    Some(GeneralGuaranteeBalance(BigDecimal(0), BigDecimal(5000.10))))
 
   implicit val appConfig = mock[AppConfig]
-  
+
   val model = GuaranteeAccountViewModel(guaranteeAccount, LocalDateTime.parse("2020-04-08T12:30"))(Helpers.stubMessages())
+  val model02 = GuaranteeAccountViewModel(guaranteeAccount02, LocalDateTime.parse("2020-04-08T12:30"))(Helpers.stubMessages())
```

Comment:
Please split this line over multiple lines as it is very long line. Also either move this val to your tests or move this to a  new trait Setup as it has been done in rest of the test classes